install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -110,6 +110,10 @@ data:
   debug-verbose: "{{ .Values.global.debug.verbose }}"
 {{- end }}
 
+{{- if .Values.global.debug.datapath }}
+  debug-datapath: "{{ .Values.global.debug.datapath }}"
+{{- end }}
```

Comment:
Shouldn't this be part of `config` rather than `global`?

Reply:
I'm not very familiar with those groupings. What's a logical difference between `config` and `global`?

Reply:
`config` is part of the settings specific to the helm chart that deploys the Cilium configmap; `global` applies settings to the entire set of helm charts we generate. Recent discussions with @cilium/helm particularly around the time of v1.8 release established that our use of Helm is a bit unusual to the detriment of operators, and one of the areas we can improve is in using more chart-local variables rather than global variables.